lib/rubocop/cop/minitest/assert_with_two_arguments.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Minitest
+      # This cop tries to detect when a user accidentally used
+      # `assert` when they meant to use `assert_equal`
```

Comment:
Can you add the period at the end of the sentence?

```suggestion
      # `assert` when they meant to use `assert_equal`.
```

Reply:
Fixed in 4b0e9ea31277b7061ab530558d4d9106c1c8cc9a